AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantListAdapter.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.helpers;
+
+import android.content.Context;
+
+import androidx.core.content.ContextCompat;
+
+import com.azure.samples.communication.calling.R;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParticipantListAdapter extends BottomCellAdapter {
```

Comment:
are we going to have only one BottomCellAdapter?  If we need multiple bottom cell in app, probably we can add the indicator to name `ParticipantListBottomCellAdapter`

Reply:
We're going to have at least three: one for Audio Selection, Participant List, and one for the additional options menu.  Here you can see the current extensions:
![image](https://user-images.githubusercontent.com/6021109/120219630-5f4b1500-c1f0-11eb-8eee-3efe01e8f576.png)